langchain/document_loaders/blockchain.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+import json
+
+from typing import List
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+ALCHEMY_API_KEY = os.environ.get("ALCHEMY_API_KEY", "docs-demo")
```

Comment:
lets not do this here, but rather inside init

Reply:
Made the update in latest commit- thanks!